commands_common/flags.go

```diff
@@ -8,7 +8,7 @@ func PromptFlag() cli.Flag {
 		Aliases:  []string{"p"},
 		Usage:    "[Optional] prompt to supply with file",
 		Value:    "",
-		Required: true,
+		Required: false,
```

Comment:
in some cases this is NOT optional, you need to pass bool flag as input to the function and based on this flag be setting it as optional or not. I commented this on the last PR. Even phrase "[Optional]" are now been shown for cases where it is NOT optional :D 

please refactor